pilot/docker/Dockerfile.proxyv2

```diff
@@ -22,23 +52,8 @@ ADD pilot-agent /usr/local/bin/pilot-agent
 ADD envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 ADD envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 ADD envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
-ADD istio-iptables.sh /usr/local/bin/istio-iptables.sh
-
-# Copy Envoy bootstrap templates used by pilot-agent
-COPY envoy_bootstrap_v2.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
-COPY envoy_bootstrap_drain.json /var/lib/istio/envoy/envoy_bootstrap_drain.json
-COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
-
-# Add CA certificates for SSL connections.
-# obtained from debian ca-certs deb using fetch_cacerts.sh
-ADD ca-certificates.tgz /
-
-RUN chmod 755 /usr/local/bin/envoy /usr/local/bin/pilot-agent
-
-# Sudoers used to allow tcpdump and other debug utilities.
-RUN useradd -m --uid 1337 istio-proxy && \
-    echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers && \
-    chown -R istio-proxy /var/lib/istio
+# istio-iptables.sh is not required for envoy
+# ADD istio-iptables.sh /usr/local/bin/istio-iptables.sh
```

Comment:
As far as we worked out, this script is never used by envoy. The container is not running in privileged mode. Therefore, running `iptables` commands should not be possible.
This is the reason, why we remove it from the docker image.
Could someone confirm?

Reply:
I think there were plans/thoughts of making the init container use proxyv2 so the image is cached and there is less startup time on pods. I would guess this is why. I would say its safe to remove since we haven't done that, but I am not sure if there is anyone depending on the script being there and doing this themselves. @sdake any ideas?

Would there be an issue with having this in the distroless image? I get that it couldn't run it, but just so we don't break someone that is using it? Although I feel a bit like https://xkcd.com/1172/

Reply:
Well, I wasn't aware of the choice to add init containers into the proxyv2 image itself.  As far as I know, the current plans indicate deprecating the init container entirely in favor of the CNI.  If the CNI cannot be used (@tiswanso are there envs where CNI won't work?), then we will need to have both models.

I still don't see a strong argument for keeping a file that is unused.  It can always be added back into the system (since this is master we are talking about) if someone has heartburn about it later...

Reply:
Yeah... the only way istio-iptables.sh would be usable would be if the container is started with net-admin.  So, for k8s that'd have to be some manual modification of the sidecar injection result.  I don't think that's likely.  For mesh expansion (VM/baremetal) I suppose it's possible there's some toolchain to set that up via the istio-proxy container.  IMO, its safe to remove.

Reply:
fair enough, I am fine either way